src/test/chain-simulator/results.cs-e2e.ts

```diff
@@ -0,0 +1,147 @@
+import axios from 'axios';
+
+const CHAIN_SIMULATOR_URL = 'http://localhost:8085';
```

Comment:
we could define these constants in a singe place

Reply:
done